docs/UserGuide/Ecosystem Integration/MapReduce TsFile.md

```diff
@@ -46,7 +46,7 @@ With this connector, you can
 ## System Requirements
 
 |Hadoop Version | Java Version | TsFile Version|
-|-------------  | ------------ |------------ |
+|:------------- : | :------------ :|:------------: |
```

Comment:
something error here.
maybe we need "blank" between ":" and "|"

Reply:
solved